config/state_machine/configs/00_moj/60_sar_internal_review/30_sar_internal_review_trigger_workflow.yml

```diff
@@ -79,6 +79,24 @@
                 unassign_from_user:
                   transition_to: awaiting_responder
 
+              responded:
+                add_message_to_case:
+                  after_transition: Workflows::Hooks#notify_responder_message_received
+                respond:
+                close:
+                  transition_to: closed
+                destroy_case:
+                edit_case:
+                  if: Workflows::Predicates#not_overturned?
```

Comment:
very minor things, for this workflow,  IMO, all those checking whether the case is overturned ico doesn't seem relevant to SAR IR

Reply:
Thanks I will remove this in my next PR - I wasn't sure what was meant by overturned so I will remove it - as the next ticket is to set up editing etc.